roads.mss

```diff
@@ -520,7 +520,10 @@ come in as well.
     line-cap: butt;
   }
   [type='railway'] {
-    line-color: fadeout(@land,50%);
+    line-width: 0;
+    [tunnel=0] {
+      line-color: fadeout(@land,50%);
+    }
```

Comment:
I would have discarded the whole `[type='railway']` block here (no more railway related stuff in this attachment).